src/graph/graph.c

```diff
@@ -1104,6 +1104,8 @@ void Graph_BulkDelete(Graph *g, Node *nodes, uint node_count, Edge *edges, uint
 			// As long as current is the same as follows.
 			while(i < edge_count - 1 && ENTITY_GET_ID(edges + i) == ENTITY_GET_ID(edges + i + 1)) i++;
 
+			// skip edges that have already been deleted
+			if(!DataBlock_GetItem(g->edges, ENTITY_GET_ID(edges + i))) continue;
```

Comment:
Consider switching to `DataBlock_ItemIsDeleted`

Reply:
These edges aren't pointers into the DataBlock; they are `SIValue` pointer types retrieved by calling `AR_EXP_Evaluate`.

Reply:
is `(edges + i)->entity` set?
if so we should be able to call `DataBlock_ItemIsDeleted((Entity*)(edges + i));`

Reply:
`entity` is set, but I don't think that has any relation to `DataBlock_ItemIsDeleted`. That function expects the bytes before the pointer to be a `DataBlockItemHeader`, which is not true in this case.